gcn/notices/core/Attitudes.schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/glowbug/Attitudes.schema.json",
+  "title": "Instrument and Spacecraft Attitudes",
+  "description": "Enables GCN producers to submit instrument and spacecraft attitude data.",
+  "type": "object",
+  "properties": {
+    "instrument_attitude": {
+      "$ref": "/schema/gcn/notices/core/AttitudeBase.schema.json",
+      "description": "Instrument attitude and rotaional rates."
+    },
+    "spacecraft_attitude": {
+      "$ref": "/schema/gcn/notices/core/AttitudeBase.schema.json",
+      "description": "Spacecraft attitude and rotaional rates."
+    }
+  }
+}
```

Comment:
Why do we need to report the attitude of the spacecraft?  The attitude of the instrument is all that matters for localizing astronomical sources.

Reply:
On second thought, the spacecraft attitude it is unnecessary. Thanks for the correction!